src/views/base.html.php

```diff
@@ -14,7 +14,26 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title><?= $title ?></title>
+    <?php
+        // Normalize the view title so it doesn't repeat the app name
+        $rawTitle = $title;
+        $sanitizedTitle = trim(
+            preg_replace(
+                [
+                    '/^\s*CronBeat\s*-\s*/i',   // strip leading "CronBeat - "
+                    '/\s*-\s*CronBeat\s*$/i',   // strip trailing " - CronBeat"
+                ],
+                '',
+                $rawTitle
+            ) ?? ''
+        );
+        // Fallback if title becomes empty
+        if ($sanitizedTitle === '') {
+            $sanitizedTitle = $isDashboard ? 'Dashboard' : 'Page';
+        }
+        $documentTitle = htmlspecialchars($sanitizedTitle, ENT_QUOTES) . ' - CronBeat';
+        ?>
+    <title><?= $documentTitle ?></title>
```

Comment:
@jetbrains-junie reduce complexity. just go by "{title} - CronBeat". do not handle `title`s containing "cronbeat".